css/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```

Comment:
Hey Juan, great job doing this project! I would like to give you some suggestions, so you can make your code more readable and professional...here we go...
script tags usually go into the head tag, please review it...

Reply:
Kindly provide a reason as to why you are requesting each of the changes. This will help make it easy to understand and implement the changes. :+1: